package.json

```diff
@@ -21,24 +23,31 @@
 	"author": "ChartIQ",
 	"license": "ISC",
 	"devDependencies": {
-		"@babel/cli": "^7.14.8",
-		"@babel/core": "^7.14.8",
+		"@babel/cli": "^7.26.4",
+		"@babel/core": "^7.26.0",
 		"@babel/plugin-syntax-dynamic-import": "^7.8.3",
-		"@babel/preset-react": "^7.14.5",
-		"babel-loader": "^8.2.2",
+		"@babel/preset-react": "^7.26.3",
+		"babel-loader": "^8.4.1",
 		"copy-webpack-plugin": "^11.0.0",
-		"css-loader": "^6.7.1",
-		"html-webpack-plugin": "^5.5.0",
-		"mini-css-extract-plugin": "^2.6.1",
+		"css-loader": "^6.11.0",
+		"html-webpack-plugin": "^5.6.3",
+		"mini-css-extract-plugin": "^2.9.2",
 		"prettier": "2.5.1",
-		"webpack": "^5.74.0",
+		"webpack": "^5.97.1",
 		"webpack-cli": "^4.10.0",
-		"webpack-dev-server": "^4.11.1"
+		"webpack-dev-server": "^4.15.2"
 	},
 	"dependencies": {
-		"@babel/preset-env": "^7.14.8",
-		"react": "^16.7.0",
-		"react-dom": "^16.7.0",
+		"@babel/preset-env": "^7.23.8",
+		"react": "^16.14.0",
+		"react-dom": "^16.14.0",
 		"react-router-dom": "^5.3.4"
+	},
+	"chartiq-dependencies-for-npm": {
+		"@chartiq/active-trader": "^9.6.0",
+		"@chartiq/core": "^9.6.0",
+		"@chartiq/react-components": "^9.1.3",
```

Comment:
```suggestion
		"@chartiq/react-components": "^9.6.2",
```

Reply:
Ah, good catch.  I know what I did wrong here for this one to not be correct.  Thanks!

Reply:
Is this just whatever the current version of the react-components is?  If so, I want to add this to the publishing script. @yorkmoy

Reply:
Yes, I believe this is the case for any normal situation I can think of.